File: cubejs/schema/Athlete.js
cube(`Athlete`, {
  sql: `SELECT * FROM public.athlete`,
  
  preAggregations: {
    // Pre-Aggregations definitions go here
    // Learn more here: https://cube.dev/docs/caching/pre-aggregations/getting-started  
  },
  
  joins: {
    
  },
  
  measures: {
    count: {
      type: `count`,
      drillMembers: [id, firstName, lastName]
    }
  },
  
  dimensions: {
    id: {
      sql: `id`,
      type: `number`,
      primaryKey: true
    },
    
    sport: {
      sql: `sport`,
      type: `string`
    },
    
    year: {
      sql: `year`,
      type: `string`
    },
    
    firstName: {
      sql: `first_name`,
      type: `string`
    },
    
    lastName: {
      sql: `last_name`,
      type: `string`
    },
    
    grade: {
      sql: `grade`,
      type: `string`
    },
    
    hometown: {
      sql: `hometown`,
      type: `string`
    },
    
    highschool: {
      sql: `highschool`,
      type: `string`
    },
    
    link: {
      sql: `link`,
      type: `string`
    }
  },
  
  dataSource: `default`
});

File: models/Athlete.py
from app import db

class Athlete(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    sport = db.Column(db.String(128))
    year = db.Column(db.String(64))
    first_name = db.Column(db.String(64))
    last_name = db.Column(db.String(64))
    grade = db.Column(db.String(64))
    hometown = db.Column(db.String(128))
    hometown_latitude=db.Column(db.Float)
    hometown_longitude=db.Column(db.Float)
    highschool = db.Column(db.String(128))
    link = db.Column(db.String(256))

    def serialize(self):
        return {
            "id": self.id,
            "sport": self.sport,
            "year": self.year,
            "first_name": self.first_name,
            "last_name": self.last_name,
            "grade": self.grade,
            "hometown": self.hometown,
            "hometown_latitude": self.hometown_latitude,
            "hometown_longitude": self.hometown_longitude,
            "highschool" : self.highschool,
            "link": self.link
        }